web/src/main/resources/META-INF/resources/webjars/common/webcert/components/wcSupportPanelManager/wcSrsPanelTab/wcSrsPanelTab.directive.scss

```diff
@@ -342,16 +230,15 @@ wc-srs-panel-tab {
       }
 
       .srsRecommendationText {
-        color: #777;
-        padding-bottom: 20px;
+        color: $wc-color-09;
       }
     }
 
-    .srs-recommendation-list-collapser, .srs-observe-list-collapser {
+    .srs-recommendation-list-collapser {
       text-decoration: underline;
       font-family: Roboto, helvetica, arial, sans-serif;
```

Comment:
Nu är jag inte insatt i srs-kraven, mell all typografi i wc är kravad utifrån ett antal typografiska stilar som man refererar till enligt  @extend %wc-typo-XX; i sin scss.

Reply:
Den har nog bara slunkit in där, behövs inte. Jag tar bort den.